ethereum/contracts/relayer/coreRelayer/CoreRelayerDelivery.sol

```diff
@@ -488,17 +502,17 @@ abstract contract CoreRelayerDelivery is CoreRelayerBase, IWormholeRelayerDelive
     }
   }
 
-  function decodeCancelled(
+  function decodeExecuteInstructionError(
```

Comment:
Perhaps we could name this `tryDecodeExecuteInstructionError`.

Reply:
done